src/Aquarius.ONE.ClientSDK/Operations/Sample/SampleCache.cs

```diff
@@ -1,4 +1,7 @@
-﻿using System;
+﻿using Newtonsoft.Json;

```

Comment:
Not sure if it's just the way the code is shown in this review but it looks like there are duplicate using statements.